daml-foundations/daml-ghc/src/DA/Daml/GHC/Compiler/Options.hs

```diff
@@ -103,14 +103,8 @@ mkOptions :: Options -> IO Options
 mkOptions opts@Options {..} = do
     baseDir <- getBaseDir
     mapM_ checkDirExists $ optImportPath <> optPackageDbs
-    let deprecatedPkgDb = baseDir </> "package-database" </> "deprecated"
-    deprecatedPkgDbExists <- Dir.doesDirectoryExist deprecatedPkgDb
-    when deprecatedPkgDbExists $
-      putStrLn "DEPRECATED: Please use fat dars instead of dalfs in tests"
-    let defaultPkgDb
-          | deprecatedPkgDbExists = deprecatedPkgDb
-          | otherwise = baseDir </> "package-database"
-    pkgDbs <- filterM Dir.doesDirectoryExist [defaultPkgDb, projectPackageDatabase]
+    let defaultPkgDb = baseDir </> "package-database"
+    pkgDbs <- filterM Dir.doesDirectoryExist [defaultPkgDb]
```

Comment:
Odd to see a filterM over a singleton... but fair enough, makes sense in context